components/tools/OmeroWeb/omeroweb/webgateway/views.py

```diff
@@ -1848,19 +1865,20 @@ def getRenderingSettings(image):
             start = ch.getWindowStart()
             end = ch.getWindowEnd()
             color = ch.getLut()
+            rev = 'r' if ch.isReverseIntensity() else ''
             if not color or len(color) == 0:
                 color = ch.getColor().getHtml()
-            chs.append("%s%s|%s:%s$%s" % (act, i+1, start, end, color))
+            chs.append("%s%s|%s:%s%s$%s" % (act, i+1, start, end, rev, color))
```

Comment:
if you went with the proposal for ':' separation then it'd need to be

`chs.append("%s%s|%s:%s:%s$%s" % (act, i+1, start, end, rev, color))`

should the first 2 %s be %d? it probably doesn't matter if they are implicitly converted.